### ISSUE TITLE
How to run localhost?

### ISSUE THREAD

**Issue by abnv418** [cloudflare/quiche#738]
Can we run localhost by having our own index.html and other pages. Just comparing it with AIOQUIC localhost ? 
Thanks

**Comment by LPardue** [comment]
More information would help. You can run all the example servers on localhost. since QUIC requires TLS, you might find that clients exhibit issues about trusting certificates. However, you can solve that problem by using trusted certifcates and hosting on corresesponding names.

In the extreme, `quiche-client` with the `--no-verify` CLI parameter will ignore certifcate problems but it is not recommended.

**Comment by abnv418** [comment]
Basically I'm trying to have my own website hosted on the local QUICHE server and then run the website using chrome/canary. I did a make on examples folder. I tried to run the server using the commands given 

cargo run --manifest-path=tools/apps/Cargo.toml --bin quiche-server -- \
      --cert tools/apps/src/bin/cert.crt \
      --key tools/apps/src/bin/cert.key

The server runs but I'm not sure where can I put my client html files in the server so I can open chrome and see the web pages using https://localhost:4433/ after starting the QUICHE server. 

How should I run the examples after running the make command in the examples folder. I want to run few webpages and see the performance on the browser. 

Thanks.
